src/main/java/com/ebay/myriad/configuration/MyriadConfiguration.java

```diff
@@ -110,6 +117,9 @@
     private String nativeLibrary;
 
     @JsonProperty
+    private String resourceManagerHost;
```

Comment:
How is this property set into the MyriadConfiguration object? I don't see this property added to myriad-config-default.yml. Am I missing something?